wiki/grips-and-tricks.md

```diff
@@ -19,16 +19,37 @@ Additionally, here's an image from the game developers about how the
 
 ![Bloq Hitbox](~@images/mapping/hitbox-from-split.jpg)
 
-## Ranking
+## Ranking Systems
 *How does global ranking work? What are performance points? What are the numbers in my brackets? Why is my PP so small?*
 
-You can find your global ranking and PP on [ScoreSaber](https://scoresaber.com/global) - here you can search for your name.
+There are 2 separate ranking systems: [ScoreSaber](https://scoresaber.com/global) and [BeatLeader](https://www.beatleader.xyz/ranking/1).
+You can find your global ranking and PP on their respective sites where you can search for your name.
+
+On **PC**, your Beatleader account is tied to your Steam account and is your steam name by default. <br>
+On **Quest**, your Beatleader account is made by you in game using the BeatLeader Mod. <br>
+You can merge your Beatleader account with your Steam account to sync scores across devices
+
 Your ScoreSaber account is tied to your Steam / Oculus username.
 
-1. Ranked songs are assigned PP (PP = performance points). The exact amount of PP gained is determined by the accuracy
+### BeatLeader
+Ranked songs are assigned 3 difficulties (stars) based on the kinds of difficulty the map presents. These 3 numbers combined with the accuracy you finished the song with determines the exact amount of raw PP gained. <br>
+The 3 difficulties represented for the purposes of PP rewards are: <br>
+
+- Pass-ibility: How hard a map is to pass. Provides a base flat PP reward<br>
+- Acc-ibility: How hard a map is to get a good accuracy. Provides PP based on your accuracy<br>
+- Tech-iness: Purly a measurement of pattern difficulty devoid of speed. Provides bonus PP based on your accuracy <br>
+
+The exact 3 difficulty numbers assigned to each map is done with an open source algorithm and AI combo developed by DzRamen and LackWiz
```

Comment:
"open source algorithm and AI combo developed" -> "open source AI algorithm and developed" ?

Reply:
how about ".... open source AI algorithm developed by ......"

Reply:
even better